From a7a6dad197b0b809fbc6e89bf9cf7b76eaea84bd Mon Sep 17 00:00:00 2001
From: Alex Coplan <alex.coplan@arm.com>
Date: Mon, 5 Oct 2020 15:49:50 +0100
Subject: [PATCH] arm: Add missing part number for Neoverse V1

This patch adds vendor and part numbers which were missing from the
initial entry for Neoverse V1 in AArch32 GCC.

gcc/ChangeLog:

	* config/arm/arm-cpus.in (neoverse-v1): Add missing vendor and
	part numbers.

Upstream-Status: Backport [9.4.0 6725de4e8376afefa7ee501ba7b31d74cbf4303d]
---
 gcc/config/arm/arm-cpus.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/gcc/config/arm/arm-cpus.in b/gcc/config/arm/arm-cpus.in
index 3c375b9a7b9..f87044fd25c 100644
--- a/gcc/config/arm/arm-cpus.in
+++ b/gcc/config/arm/arm-cpus.in
@@ -1370,6 +1370,8 @@ begin cpu neoverse-v1
   architecture armv8.4-a+fp16
   option crypto add FP_ARMv8 CRYPTO
   costs cortex_a57
+  vendor 41
+  part 0xd40
 end cpu neoverse-v1
 
 # Armv8.5 A-profile Architecture Processors
